hw/dv/tools/dvsim/xcelium.hjson

```diff
@@ -119,6 +119,27 @@
                      "-init {xcelium_common_excl_file}",
                      " {eval_cmd} echo {xcelium_cov_refine_files} | sed -E 's/(\\S+)/-load_refinement \\1/g' "]
 
+  cov_unr_dir:     "{scratch_path}/cov_unr"
+
+  // not needed we can reuse the last build snapshot
+  cov_unr_build_opts:[]
+  cov_unr_build_cmd:[]
+  
+  cov_unr_run_cmd: "{job_prefix} xrun"
+
+               
+  cov_unr_run_opts: ["-unr",
+                     "-jg",
```

Comment:
-use jason gold

Reply:
fixed

Reply:
Sorry, did you mean JasperGold?

Reply:
@sriyerg oops yes I did

Reply:
@rasmus-madsen, you merged this PR without addressing this comment.